src/identity/mod.rs

```diff
@@ -154,6 +157,22 @@ impl Identity {
             stream: "mock-stable".to_string(),
         }
     }
+
+    /// Validate the group label value in current identity.
+    fn validate_group_label(&self) -> Fallible<()> {
+        static VALID_GROUP: &str = "^[a-zA-Z0-9.-]+$";
```

Comment:
Do we want to support groups starting with `.`, `-` or a number? i would recommend we only allow letters for the first char.

Reply:
I wouldn't disallow them (especially numbers only), but not strong opinion either way. However, this is coming from an already-defined regex https://coreos.github.io/zincati/development/fleetlock/protocol/#body. I should probably add the reference in comments.

Reply:
I've just added the specs reference in the docstring, but kept the regex as-is to match previous docs.